tools/cbmc/proofs/ARP/ARP_OutputARPRequest_buffer_alloc1/README.md

```diff
@@ -0,0 +1,27 @@
+Abstracting away vPortEnterCritical, vPortExitCritical, vAssertCalled,
```

Comment:
I would lead with a simple statement (actually in all three top-level harness directories) that you are proving the memory safety of OutputARPRequest with the first buffer allocator assuming that concurrency (critical sections, task management) and tracing are memory safe, and that the third-party low-level buffer management is correct.  And in more detail...

Similar for the readme for the second buffer allocator.

And in the readme for the third proof (what was the initial proof) say that you are experimenting with potential errors in the third-party allocators.